de.html

```diff
@@ -909,8 +909,8 @@
 
 <panel w=600 h=110>
 <words w=600 x=-15 no-bg>
-	Jedes Spiel dauert 20-30 Minuten – so lange wie eine TV-Serie.
-	Dafür kannst du dich ein <i>Leben lang</i> an alles mögliche erinnern!
+	Jedes tägliches Spiel dauert 20-30 Minuten. Anstatt eine Serie im TV zu schauen,
+	kannst du dich ein <i>Leben lang</i> an alles mögliche erinnern!
```

Comment:
```suggestion
	kannst du dich ein <i>Leben lang</i> an alles Mögliche erinnern!
```
https://www.korrekturen.de/wortliste/alles_moegliche.shtml